packages/cli/src/commands/inspect.ts

```diff
@@ -98,12 +134,26 @@ export default async function main(client: Client) {
     `Fetching deployment "${deploymentIdOrHost}" in ${chalk.bold(contextName)}`
   );
 
+  const until = Date.now() + ms(timeout);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏`ms` only once

This isn't a performance issue, I'm sure. It just seems a bit weird that we invoke `ms(timeout)` twice here.

I notice that we do a similar thing in `vc rollback`. I'd rather parse the args closer to where we pluck them from the global context and pass along parsed args through the system.

Reply:
Fixed.